basic_datatypes.ipynb

```diff
@@ -2085,9 +2083,9 @@
    "outputs": [],
    "source": [
     "%%ipytest\n",
-    "import typing\n",
-    "T = typing.Typevar(\"T\")\n",
-    "def solution_dict_return_value(my_dict: dict[typing.Hashable, T], key: typing.Hashable) -> typing.Optional[T]:\n",
+    "from typing import TypeVar, Hashable, Any\n",
+    "\n",
+    "def solution_dict_return_value(my_dict: dict[Hashable, Any], key: Hashable) -> Any:\n",
     "    # Your code starts here\n",
```

Comment:
Why did you remove the typevar? Now the type is "too generic"?

Reply:
I thought the value of a dictionary could be any Python object, which I've done by setting the `Any` type. I do not understand the exact problem here.

The original implementation had a bug that @despadam reported above, so I looked around and this is what I found as the solution.

Reply:
It is correct, but if you use a type variable, then the function has a more specific return type. It does not really matter for the moment, so I suggest to merge anyway

Reply:
It also looks ok to me and solves the bugs, so I think that since these exercises are anyway trivial, there is no need to worry about typing now